dspace-api/src/main/java/org/dspace/app/sitemap/GenerateSitemaps.java

```diff
@@ -150,14 +180,10 @@ public static void main(String[] args) throws Exception {
      * @throws IOException  if IO error
      *                      if IO error occurs.
      */
-    public static void generateSitemaps(boolean makeHTMLMap,
-                                        boolean makeSitemapOrg) throws SQLException, IOException {
-        String sitemapStem = configurationService.getProperty("dspace.ui.url")
-            + "/sitemap";
-        String htmlMapStem = configurationService.getProperty("dspace.ui.url")
-            + "/htmlmap";
-        String handleURLStem = configurationService.getProperty("dspace.ui.url")
-            + "/handle/";
+    public static void generateSitemaps(boolean makeHTMLMap, boolean makeSitemapOrg) throws SQLException, IOException {
+        String sitemapsEndpoint = configurationService.getProperty("sitemap.path", "sitemaps");
+        String uiURLStem = configurationService.getProperty("dspace.ui.url");
+        String sitemapStem = uiURLStem + "/" + sitemapsEndpoint + "/sitemap";
```

Comment:
Minor change required here.  Based on updates to https://github.com/DSpace/dspace-angular/pull/825 , this `sitemapStem` should be updated to just be:
`String sitemapStem = uiURLStem + "/sitemap";`

The reason is that, at the UI level, the `sitemapsEndpoint` has no meaning. It's only valid for the REST API.  This is needed, to ensure that the sitemap files are available at the root level of the UI (e.g. http://localhost:400/sitemap_index.xml , http://localhost:400/sitemap0.xml)

Reply:
@tdonohue Applied fix; https://github.com/DSpace/DSpace/pull/2858/commits/4b86f331d43041d3991124534c4c546068d75546

Reply:
Thanks @MarieVerdonck !  I'll give this one last test then and merge it assuming it all works (and it should).